js/app.js

```diff
@@ -64,29 +62,22 @@ function createRandomImages(){
         }
     }
 }
```

Comment:
While creating an new element for each image works just fine, it is more efficient to just change the `src` of an `img` already on the page. It takes time to create and destroy elements.